### ISSUE TITLE
Python3 - impossible to use unicode string in get_vector function

### ISSUE THREAD

**Issue by pommedeterresautee** [salestock/fastText.py#10]
```python
    def get_vector(self, word):
        word_bytes = bytes(word, 'ascii')
        return self.fm.getVectorWrapper(word_bytes)
```
It is impossible to ask for a vector with unicode spelled word because of the conversion in bytes set to ascii

**Comment by pyk** [comment]
@pommedeterresautee ahh nice catch. Hotfix is in progress. thanks for the report!

**Comment by pyk** [comment]
@pommedeterresautee fix is available at `v0.5.18` or #11 , need to setup the CI first (again) before merging

Now you can do like the following:

```python
model.get_vector('Καλημέρα')
```
